### PR TITLE
Feature: Venue Owner management

### PR DESCRIPTION
**Pull request by alex-go-nguyen**
# Issue

- Chart APIs

## Summary

- Fix Roles Guard

### Media showcase

<Screenshot / video>

- [x] If an existing feature was modified, attached media showcases both before and after the change(s).
- [x] If configuration options are introduced, attached media showcases both on and off state(s).

### Author

- [x] I have reviewed my own code (all files marked as viewed in GitHub).
- [x] I have tested (via the postman) each of the interfaces/features related to the files changed/added.

### CODE REVIEW

**Comment by michael-do-goldenowl** [comment]
@WithMaverick

File: src/cloudinary/cloudinary.service.ts
Comment (michael-do-goldenowl) [inline src/cloudinary/cloudinary.service.ts:1]: @WithMaverick

**Comment by michael-do-goldenowl** [comment]
@WithMaverick

**Comment by alex-go-nguyen** [comment]
@WithMaverick

**Comment by alex-go-nguyen** [comment]
@WithMaverick

**Comment by alex-go-nguyen** [comment]
@WithMaverick

**Comment by alex-go-nguyen** [comment]
@WithMaverick

**Comment by alex-go-nguyen** [comment]
@WithMaverick

**Comment by alex-go-nguyen** [comment]
@WithMaverick

**Comment by alex-go-nguyen** [comment]
@WithMaverick

**Comment by alex-go-nguyen** [comment]
@WithMaverick

**Comment by alex-go-nguyen** [comment]
@WithMaverick

**Comment by alex-go-nguyen** [comment]
@WithMaverick

**Comment by alex-go-nguyen** [comment]
@WithMaverick

**Comment by alex-go-nguyen** [comment]
@WithMaverick

**Comment by alex-go-nguyen** [comment]
@WithMaverick

**Comment by alex-go-nguyen** [comment]
@WithMaverick

**Comment by alex-go-nguyen** [comment]
@WithMaverick

**Comment by alex-go-nguyen** [comment]
@WithMaverick

**Pull request merged**